.github/workflows/buildpack-ubuntu2004.yml

```diff
@@ -0,0 +1,33 @@
+name: buildpack:ubuntu2004
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/buildpack-ubuntu2004.yml'
+      - 'scripts/docker/buildpack/Dockerfile.ubuntu2004'
+jobs:
+  build:
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      DOCKER_REPOSITORY: docker.pkg.github.com/${{ github.repository }}
+      IMAGE_NAME: buildpack
```

Comment:
Since you did make this configurable ;-).
The canonical name (and the name most images are based on) would be ``buildpack-deps`` wouldn't it? And you're dropping the original ``solidity-`` prefix since that's implicit in the repo URL ``docker.pkg.github.com/solidity``?
That being said/asked, I don't really mind the actual name, as long as it makes clear that it's not a solc release image...

Reply:
I think its better to use `buildpack-deps`.